pulpcore/cli/core/pulpexport.py

```diff
@@ -0,0 +1,107 @@
+from typing import List
+import click
+from pulpcore.cli import main
+
+RepoIdentifier = click.Tuple([str, str])
```

Comment:
:+1:

Reply:
click and mypy and typing are mad at each other, I have work to do.